packaging/docker/run.sh

```diff
@@ -46,9 +39,16 @@ if [ -n "${PGID}" ]; then
   usermod -a -G "${PGID}" "${DOCKER_USR}" || echo >&2 "Could not add netdata user to group docker with ID ${PGID}"
 fi
 
-if mountpoint -q /etc/netdata && [ -z "$(ls -A /etc/netdata)" ]; then
+if mountpoint -q /etc/netdata && [ -z "$(ls -A /etc/netdata | grep -v netdata.conf)" ]; then
   echo "Copying stock configuration to /etc/netdata"
-  cp -a /etc/netdata.stock/. /etc/netdata
+  cp -an /etc/netdata.stock/. /etc/netdata
```

Comment:
I think we need only 2 changes:

- remove `[ -z "$(ls -A /etc/netdata)" ]` check
- add the `n` option to `cp`.

That is it, should work without moving code around.

Reply:
You're right. This will allow also to have other config files configured at startup.
Changes applied.